lib/MTT/Test/Build/Trivial.pm

```diff
@@ -161,11 +165,12 @@ sub Build {
     # linking simple "use mpi_f08" applications
 
     if ($mpi_install->{usempif08_bindings} and $languages_hash->{"usempif08"}) {
+        $f08flags = $fflags if (!defined($f08flags));
         Debug("Test compile/link sample \"use mpi_f08\" MPI application\n");
-        $x = _do_compile("$mpif08 $fflags", "hello_usempif08.f90", "hello_usempif08");
+        $x = _do_compile("$mpif08 $f08flags", "hello_usempif08.f90", "hello_usempif08");
```

Comment:
If we're not going to get rid of fflags, then you should have *both* `$fflags` and `$usempif08_flags` in there:

```perl
$x = _do_compile("$mpif08 $fflags $usempif08_flags", ...
```